css/login-signup.css

```diff
@@ -0,0 +1,91 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+
+@import url("/css/navbar.css");
+@import url("/css/footer.css");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  overflow-x: hidden;
+  color: var(--text-color);
+}
+
+/* Login */
+
+.login-container {
+  margin: 2rem;
+}
+
+.card-wrapper {
+  margin: auto;
+  padding: 1rem;
+  max-width: 400px;
+}
+
+.login-container .card-heading {
+  font-size: var(--heading-2);
+  font-weight: bold;
+  text-align: center;
+}
+
+/* Form */
+
+.login-container .input-group {
+  margin: 0.5rem 0;
+}
+
+.login-container label {
+  font-size: 0.9rem;
+  line-height: var(--line-ht-5);
+}
+
+.login-container input {
+  font-size: 1rem;
+  padding: 0.25rem 0.5rem;
+}
+
+.login-container .input-checkbox {
+  display: flex;
+  flex-direction: row;
```

Comment:
flex-direction is row by default

Reply:
Yes true, but I have written it to override the direction applied by another class (imported from Component Library) which was `column`.
Thank you!